acq_stat_reports/acq_stat_reports.py

```diff
@@ -5,8 +5,10 @@
 import argparse
 import json
 import os
+import warnings
 from pathlib import Path
 
+import agasc
 import jinja2
 import matplotlib.pyplot as plt
```

Comment:
Does it also make sense to use Tom's new plot-script convention of 
```
# Force non-interactive Agg backend for matplotlib in the context of this script
os.environ["MPLBACKEND"] = "Agg"
```
?

Reply:
Where is that used?

The problem is that this needs to be set before matplotlib is imported. There was a time when `matplotlib.use('agg')` had to be called before importing matplotlib, but I just checked and it works even after I already created another figure.

I think that means the correct way is to use `matplotlib.use('agg')` in `main()`, not at module level.

Reply:
@taldcroft suggested it to me for https://github.com/sot/ska_trend/blob/26441a05574e8f593db5b51fa93c9fdb80c1a9fe/ska_trend/wrong_box_anom/wrong_box_anom.py#L15 with the idea that for these codes that are meant to make plots etc and aren't meant to be imported that it would be simplest to do via the env var.

Reply:
I just tried using `matplotlib.use('agg')` in main. That does not prevent the usual `RuntimeWarning: More than 20 figures have been opened.` I think it is good to add it. This would prevent the script from popping an undesired window.

As for the `close_figures` option/argument, this has an effect only when running inside ipython. If you call the function from ipython, it briefly opens a window and then closes it.